rollup.config.js

```diff
@@ -1,14 +1,29 @@
-const { defineConfig } = require("rollup");
-const { nodeResolve } = require("@rollup/plugin-node-resolve");
-const typescript = require("@rollup/plugin-typescript");
+import { defineConfig } from "rollup";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import scss from "rollup-plugin-scss";
+import { copy } from "@web/rollup-plugin-copy";
 
-module.exports = () =>
-  defineConfig({
-    input: "src/ose.js",
-    output: {
-      dir: "dist/",
-      format: "es",
-      sourcemap: true,
-    },
-    plugins: [nodeResolve(), typescript()],
-  });
+const staticFiles = ["lang", "packs", "templates"];
+
+export default defineConfig({
+  input: "src/ose.js",
+  output: {
+    dir: "dist/",
+    format: "es",
+    sourcemap: true,
+  },
+  plugins: [
+    nodeResolve(),
+    typescript(),
+    scss({
```

Comment:
When used from `build:watch`, is this going to trigger a reload of the page for updates to our CSS?

I'd be a fan of getting it to hot reload instead, but I get if we can't.

Reply:
Remove livereload for now

Reply:
Update pushed, we have in-place css reloading now